client/ide/lib/workspace/panes/idedrawingpane.coffee

```diff
@@ -110,12 +111,12 @@ module.exports = class IDEDrawingPane extends IDEPane
 
   init: ->
 
-    @$canvas = @canvas.getDomElement()
+    @$canvas = $('canvas.drawing')
```

Comment:
this is not a correct solution, but let's merge this one for now. we need to revert this later. this selector looks dangerous to me.